microsetta_public_api/api/taxonomy.py

```diff
@@ -44,6 +45,13 @@ def summarize_group(body, resource):
     return _summarize_group(sample_ids, resource, taxonomy_repo)
 
 
+def get_empress(dataset, resource):
+    taxonomy_repo = _get_taxonomy_repo(dataset)
+    taxonomy_model = taxonomy_repo.model(resource)
+    empress_model = Empress(taxonomy_model.bp_tree)
+    return empress_model._to_dict()
```

Comment:
Seems weird to call a private member method of `Empress`?

Reply:
Opened an issue regarding this https://github.com/biocore/empress/issues/470

Reply:
Based on the discussion, the `_to_dict` method does not currently guarantee that changes to the result of `empress_model._to_dict()` will not change the state of `empress_model`. But this should be fine with the pattern here. The issue is open to create a public version of the function.

Reply:
Thanks!!